src/main/java/equipment/electronic/InternetPC.java

```diff
@@ -0,0 +1,13 @@
+package equipment.electronic;
+
+import equipment.ElectronicEquipment;
+
+public class InternetPC extends ElectronicEquipment {
+    public InternetPC(String id, String ip) {
+        super(id, ip);
+    }
+
+    public String toString() {
+        return "Internet PC - " + ip;
```

Comment:
code duplication- if every `ElectronicEquipment` will need to append the `IP` to the equipment's description, make this logic part of the base class